src/lib-components/Page/index.tsx

```diff
@@ -34,26 +36,37 @@ export function Page({
   footer,
   children,
   centralized = false,
+  justifyContent = false,
+  alignItems = false,
   column = false,
   haveToast = false,
   components,
   createNavbarContext = true
 }: PageProps) {
   const navbarRef = useRef<HTMLDivElement>(null);
   const footerRef = useRef<HTMLDivElement>(null);
+  const childrenWrapper = useRef<HTMLDivElement>(null);
+  const firstRender = useRef<boolean>(true);
+
+  const [childrenHeight, setChildrenHeight] = useState<number>(0);
   const [dimensions, setDimensions] = useState<Dimensions>({
     navHeight: 0,
     footHeight: 0
   });
 
-  const firstRender = useRef<boolean>(true);
+  useEffect(() => {
+    setChildrenHeight(
+      childrenWrapper.current ? childrenWrapper.current.offsetHeight : 0
+    );
+  }, [childrenWrapper.current?.offsetHeight]);
```

Comment:
useRef don't trigger the useEffect, hence, the children height will not change. A change to useCallback would be more appropriate